packages/template-retail-react-app/config/default.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
         url: {
             site: 'path',
             locale: 'path',
-            showDefaults: true
+            showDefaults: true,
+            interpretPlusSignAsSpace: false
```

Comment:
Ok.. I actually change my mind about this. Because this will become part of the API for the configuration file. I think it's best that this lives under `app.search` and not `app.url` .. my bad for making that suggestion earlier.